grakn-engine/src/test/java/ai/grakn/engine/controller/SystemControllerTest.java

```diff
@@ -98,17 +104,19 @@ public void whenCallingPutKBEndpoint_Return200_AndConfigInBody() throws JsonProc
     }
 
     @Test
-    public void whenCallingPutKBEndpointOnNonExistentKeyspace_KeyspaceAppearsInList() {
+    public void whenCallingPutKBEndpointOnNonExistentKeyspace_KeyspaceAppearsInList() throws IOException {
         RestAssured.put("/kb/myks");
-        when().get("/kb").then().body("", hasItem("myks"));
+        String content = when().get("/kb").thenReturn().body().asString();
+        Keyspaces keyspaces = objectMapper.readValue(content, Keyspaces.class);
+        assertThat(keyspaces.keyspaces(), not(empty()));
```

Comment:
Apparently Rest Assured should handle this for us: https://github.com/rest-assured/rest-assured/wiki/Usage#object-mapping

```
Keyspaces keyspaces = get("/kb").as(Keyspaces.class);
```